### DISCUSSION TITLE
Gemini输出的文案格式没有被正确识别[Bug]

### DISCUSSION THREAD

**Discussion by coulsontl** [lobehub/lobehub#1847] [General Question] [answered]
### 💻 系统环境

Ubuntu

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述

如题，Gemini输出的文案格式没有被正确识别，后端用的是OneApi部署的中转。现在显示的样式如下，都是*号： <img width="525" alt="image" src="https://github.com/lobehub/lobe-chat/assets/149155411/d12188b2-b5c9-424d-8108-8d147af4bd8f">

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这是 gemini 输出的文本没有符合标准的 markdown 规范吧？ `** **` 代表的强调语法，是需要前后有空格的。

[a comment by an automated account is not shown]

**Comment by coulsontl** [comment]
> 这是 gemini 输出的文本没有符合标准的 markdown 规范吧？ `** **` 代表的强调语法，是需要前后有空格的。

大佬能否兼容一下，不然Gemini渠道用着太别扭了

[a comment by an automated account is not shown]

**Comment by MapleEve** [comment, marked as answer]
> > 这是 gemini 输出的文本没有符合标准的 markdown 规范吧？ `** **` 代表的强调语法，是需要前后有空格的。
> 
> 大佬能否兼容一下，不然Gemini渠道用着太别扭了

你在 prompt 里面加上字体加粗的 Markdown必须符合标准格式就好了，否则他输出的不是Markdown 规范谁也没办法兼容……

[a comment by an automated account is not shown]
